default-theme.html

```diff
@@ -10,30 +10,51 @@
 
 <link rel="import" href="../polymer/polymer.html">
 
+<!-- Taken from https://www.google.com/design/spec/style/color.html#color-ui-color-application -->
+
 <style is="custom-style">
 
   :root {
-
-    --dark-primary-color: #303f9f;
-
-    --default-primary-color: #3f51b5;
-
-    --light-primary-color: #c5cae9;
-
-    --text-primary-color: #ffffff;
-
-    --accent-color: #ff4081;
-
-    --primary-background-color: #ffffff;
-
-    --primary-text-color: #212121;
-
-    --secondary-text-color: #757575;
-
-    --disabled-text-color: #bdbdbd;
-
-    --divider-color: #e0e0e0;
-
+    /*
+     * Light background theme
+     */
+    --light-theme-background-color: #ffffff;
+    --light-theme-base-color: #000000;
+    --light-theme-text-color: #212121;
+    --light-theme-secondary-color: #737373;  /* for secondary text and icons */
+    --light-theme-disabled-color: #9b9b9b;  /* disabled/hint text */
+    --light-theme-divider-color: #dbdbdb;
+
+    /*
+     * Dark background theme
+     */
+    --dark-theme-background-color: #212121;
+    --dark-theme-base-color: #ffffff;
+    --dark-theme-text-color: #ffffff;
+    --dark-theme-secondary-color: #bcbcbc;  /* for secondary text and icons */
+    --dark-theme-disabled-color: #646464;  /* disabled/hint text */
+    --dark-theme-divider-color: #3c3c3c;
+
+    /*
+     * Primary and accent colors. Also see color.html for more colors.
+     */
+    --light-primary-color: #c5cae9;  /* --paper-indigo-100 */
+    --default-primary-color: #3f51b5;  /* --paper-indigo-500 */
```

Comment:
@notwaldorf get this colors from [here](https://www.google.com/design/spec/style/color.html#color-ui-color-application):

<img width="782" alt="screen shot 2015-12-16 at 16 49 24" src="https://cloud.githubusercontent.com/assets/1007051/11845624/0ab098ac-a415-11e5-8e03-4ff3b02383c6.png">